packages/forms-web-app/src/dynamic-forms/question.js

```diff
@@ -318,9 +318,12 @@ class Question {
 	 * }>}
 	 */
 	formatAnswerForSummary(sectionSegment, journey, answer, capitals = true) {
+		const isAnswerProvided = answer !== null && answer !== undefined && answer !== '';
```

Comment:
empty strings are also falsy, couldn't you just handle zero like this and store a number?

Reply:
Not quite sure what you mean here. 0 is/ was being passed/ saved as a number. It was being saved in to the journey and db as 0 as expected but on the task list and question page 0 wouldnt be shown. The task list would show 'Not started' and question page would have an empty field as if it hadnt been answered but looking in the logs it was in the response

In dynamic-components/number-entry/question.js 0 was being passed in as a number but in order for it to display the only way I found was converting it to string:

> const persistedAnswer =
			answer !== undefined && answer !== null && answer === 0
				? `${answer}`
				: answer !== undefined && answer !== null
				? answer
				: '';